packages/form-js-viewer/src/render/components/form-fields/Radio.js

```diff
@@ -1,4 +1,5 @@
 import { useRef } from 'preact/hooks';
+import isDeepEqual from 'lodash/isEqual';
```

Comment:
Let's not rename it unless it's necessary

```suggestion
import isEqual from 'lodash/isEqual';
```